src/Components/Form/questions.js

```diff
@@ -1,68 +1,62 @@
 export const questionSet = [
     {
-        question: 'Does your group include kids?', 
-        choices: ['yes', 'no'],
-        activities: [],
-        answerType: 'default'
-    }, 
-    {
-        question: 'Which activities excite you right now?',
+        question: 'Which activity excite you right now?',
         choices: ['movies', 'board games', 'music', 'card games', 'podcasts'],
-        activities: [],
+        activit: '',
         answerType: 'default'
     },
     {
       question: 'How old is too old?',
       choices: ['2010\'s', '2000\'s','1990\'s', '1980\'s'],
-      activities: ['music'],
+      activity: 'music',
```

Comment:
This makes more sense to have a string and singular naming convention. I noticed a ton of inconsistencies like this in our database so I have made a point to fix that in our backend

Reply:
Thank you!

Reply:
agree,  this makes more sense. Thanks